crates/core/src/factories/core_space/test.rs

```diff
@@ -93,6 +95,8 @@ async fn space_local_agent_join_leave() {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn space_notify_send_recv() {
+    enable_tracing();
```

Comment:
Do you want to keep this tracing? I don't bother about having tracing in tests in general but it seems to be the only change in this file.

Reply:
I do, ideally it'd be enabled in all tests. I enabled tracing in two tests that I was debugging while making this change and it was really useful to be able to see what they're doing.